translations/kubectl/de_DE/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3191 @@
+"Language-Team: Steffen Schmitz <steffenschmitz@hotmail.de>\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+
+#: pkg/kubectl/cmd/create_clusterrolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a ClusterRoleBinding for user1, user2, and group1 using the "
+"cluster-admin ClusterRole\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin --clusterrole=cluster-"
+"admin --user=user1 --user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t # Erstellt ein ClusterRoleBinding für user1, user2 und group1 mit der "
+"ClusterRole cluster-admin\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin --clusterrole=cluster-"
+"admin --user=user1 --user=user2 --group=group1"
+
+#: pkg/kubectl/cmd/create_rolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a RoleBinding for user1, user2, and group1 using the admin "
+"ClusterRole\n"
+"\t\t  kubectl create rolebinding admin --clusterrole=admin --user=user1 --"
+"user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t  # Erstellt ein RoleBinding für user1, user2 und group1 mit der "
+"ClusterRole admin\n"
+"\t\t  kubectl create rolebinding admin --clusterrole=admin --user=user1 --"
+"user=user2 --group=group1"
+
+#: pkg/kubectl/cmd/create_configmap.go:44
+msgid ""
+"\n"
+"\t\t  # Create a new configmap named my-config based on folder bar\n"
+"\t\t  kubectl create configmap my-config --from-file=path/to/bar\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with specified keys instead "
+"of file basenames on disk\n"
+"\t\t  kubectl create configmap my-config --from-file=key1=/path/to/bar/file1."
+"txt --from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with key1=config1 and "
+"key2=config2\n"
+"\t\t  kubectl create configmap my-config --from-literal=key1=config1 --from-"
+"literal=key2=config2"
+msgstr ""
+"\n"
+"\t\t  # Erstellt eine neue ConfigMap mit dem Namen my-config basierend auf "
+"dem Ordner bar\n"
+"\t\t  kubectl create configmap my-config --from-file=path/to/bar\n"
+"\n"
+"\t\t  # Erstellt eine neue ConfigMap mit dem Namen my-config mit den "
+"angegebenen Keys statt des Dateinamens auf der Festplatte.\n"
+"\t\t  kubectl create configmap my-config --from-file=key1=/path/to/bar/file1."
+"txt --from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Erstellt eine neue ConfigMap mit dem Namen my-config mit key1=config1"
+" und key2=config2\n"
+"\t\t  kubectl create configmap my-config --from-literal=key1=config1 --from-"
+"literal=key2=config2"
+
+#: pkg/kubectl/cmd/create_secret.go:135
+msgid ""
+"\n"
+"\t\t  # If you don't already have a .dockercfg file, you can create a "
+"dockercfg secret directly by using:\n"
+"\t\t  kubectl create secret docker-registry my-secret --docker-"
+"server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-"
+"password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL"
+msgstr ""
+"\n"
+"\t\t  # Wenn Du noch keine .dockercfg Datei hast, kannst du direkt ein "
```

Comment:
Shouldn't we use passive instead of "Du"?

Reply:
I was also not sure how to handle this. I will rewrite it in passive.